FiveCalls/FiveCalls/IssueDetailViewController.swift

```diff
@@ -196,7 +196,7 @@ extension IssueDetailViewController : EditLocationViewControllerDelegate {
     }
     
     func editLocationViewController(_ vc: EditLocationViewController, didUpdateLocation location: UserLocation) {
-        issuesManager.fetchIssues(.top(location)) { result in
+        issuesManager.fetchIssues(false, location: location) { result in
```

Comment:
My first attempt was just both cases of `Query` had the `UserLocation` associated type. Was easy to swap to that, but it feels funky. I'll go back to the plain enum here.